front/components/vaults/VaultCreateFolderModal.tsx

```diff
@@ -110,7 +110,7 @@ export default function VaultCreateFolderModal({
       <Page variant="modal">
         <div className="w-full">
           <Page.Vertical sizing="grow">
-            <Page.H>Name</Page.H>
+            <Page.SectionHeader title="Name" />
```

Comment:
Here and below: just fixing a title size on the modal to create a Folder (matching the figma).